### DISCUSSION TITLE
What does "terms_and_conditions.rst" mean, legally?

### DISCUSSION THREAD

**Discussion by XeCycle** [Seagate/cortx#588] [Q&A] [answered]
Hi,

I suppose Apache License 2.0 allows me to use the software commercially, for whatever application.  But `terms_and_conditions.rst` says "Users should use this only for testing purposes and should not store any critical data on it" -- how does this statement apply?  Is the file just a disclaimer like "Seagate will not provide support for CORTX community", or is it a legal restriction on usage of the software?

**Comment by johnbent** [comment, marked as answer]
Thank you for your question and interest @XeCycle .  It's a disclaimer about support; it is not a legal restriction on usage of the software.
